pgjdbc/src/main/java/org/postgresql/util/PGInterval.java

```diff
@@ -468,6 +471,41 @@ public void scale(int factor) {
     setSeconds(factor * getSeconds());
   }
 
+  /**
+   * Creates a new PGIntervall with the given period and duration values.
+   *
+   * @param period the Period part (years, months and days) of the new interval.
+   * @param duration the Duration part (hours, minutes and seconds) of the new interval.
+   * @return a PGInterval with this filled values from the passed period and duration.
+   */
+  public static PGInterval from(Period period, Duration duration) {
```

Comment:
Java's convention is `Duration.of`, `Period.of`. Should we go for `PGInterval.of` as well?

Reply:
Sure, changed to `of`.